src/views/map.c

```diff
@@ -2183,6 +2184,55 @@ static gboolean _view_map_remove_pin(const dt_view_t *view, OsmGpsMapImage *pin)
   return osm_gps_map_image_remove(lib->map, pin);
 }
 
+static void _track_add_point(OsmGpsMapTrack *track, OsmGpsMapPoint *point, OsmGpsMapPoint *prev_point)
+{
+  float lat, lon, prev_lat, prev_lon;
+  osm_gps_map_point_get_degrees(point, &lat, &lon);
+  osm_gps_map_point_get_degrees(prev_point, &prev_lat, &prev_lon);
+  double d, delta;
+  gboolean short_distance = TRUE;
+  if (
```

Comment:
no space after `if` and should be indented as:

```
if(fabs(lat - prev_lat) > DT_MINIMUM_ANGULAR_DELTA_FOR_GEODESIC
   || fabs(lon - prev_lon) > DT_MINIMUM_ANGULAR_DELTA_FOR_GEODESIC)
{
```

Reply:
done